stix_shifter_modules/azure_sentinel/requirements.txt

```diff
@@ -0,0 +1,4 @@
+azure-identity
+azure-monitor-query==1.0.2
+pandas==1.1.5
+regex==2022.7.9
```

Comment:
what's the use of this regex package? I would recommend to use the standard regex rather than this wrapper package.

Reply:
This regex package is required as I am looking for recursive patterns in string response. The recursive pattern is not supported in the standard regex